includes/TRAITS/Assets.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Trait Assets
+ *
+ * @package   Google\Web_Stories\TRAITS
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\TRAITS;
+
+/**
+ * Trait Assets
+ *
+ * @package Google\Web_Stories\TRAITS
+ */
+trait Assets {
+	/**
+	 * Helper method to load assets based on a handle.
+	 *
+	 * @param string $handle Handle of script.
+	 * @param array  $extra_dependencies Array of extra dependencies.
+	 */
+	public function load_assert( $handle, array $extra_dependencies = [] ) {
```

Comment:
Typo: `assert` -> `asset`

Reply:
Should we rename this to `load_script`? And create `load_style` in addition?

Reply:
I have changed to `load_asset`. This one method now loads css and js.

Reply:
What if we ever want to load just CSS, no JS?

Also, as per my other comment we need to make it possible to just _register_ assets, not _enqueue_.

Wouldn't be easier if we'd have a a structure like this:

```php
private function get_asset_metadata( $handle ) {
	$asset_file   = WEBSTORIES_PLUGIN_DIR_PATH . 'assets/js/' . $script_handle . '.asset.php';
	$asset        = is_readable( $asset_file ) ? require $asset_file : [];
	if ( ! $asset['dependencies'] ){
		$asset['dependencies'] = [];
	}
	if ( ! $asset['version'] ){
		$asset['version'] = WEBSTORIES_VERSION;
	}

	return $asset;
}

public function register_script( $handle, $extra_dependencies ) {
	$metadata = $this->get_asset_metadata(...);
	wp_register_script(...);
	wp_set_script_translations( $handle, 'web-stories' );
}

public function enqueue_script( $handle, $extra_dependencies ) {
	$this->register_script( $handle, $extra_dependencies );
	wp_enqueue_script( $handle );
}

public function register_style( $handle, $extra_dependencies ) {
	$metadata = $this->get_asset_metadata(...);
	wp_register_style(...);
}

public function enqueue_style( $handle, $extra_dependencies ) {
	$this->register_style( $handle, $extra_dependencies );
	wp_enqueue_style( $handle );
}
```